localstack-core/localstack/services/events/event_bus.py

```diff
@@ -87,7 +99,14 @@ def revoke_put_events_permission(self, statement_id: str):
             ]
             self.event_bus.last_modified_time = datetime.now(timezone.utc)
 
-    def _pars_statement(self, statement_id, action, principal, resource_arn, condition):
```

Comment:
typo: parse?

Reply:
fixed in: 1620701d4de69e12506b5fab2c22c37cf65abfac